src/sidebar/store/modules/direct-linked-group.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const util = require('../util');
+
+function init() {
+  return {
+    /**
+     * Indicates that an error occured in retrieving/showing the direct linked group.
+     * This could be because:
+     * - the group does not exist
+     * - the user does not have permission
+     * - the group is out of scope for the given page
+     * @type {boolean}
+     */
+    directLinkedGroupFetchFailed: false,
```

Comment:
I have mixed feeling about this one. Yes, it is groups related so it could possibly go in the groups module. I'm not sure that's enough of an argument to put it there though. The groups module is already very large, it consistently holds the state of the groups and the focused group and while the direct-linked group has influence on the focused group, I'm not sure tracking it's error state belongs there. The fetch failed state is specific to the direct-linked group, not to all groups. In keeping with the current placement in the store modules, the state of direct-linked/selected annotation(s) are not stored in the annotations module, they are stored in the selection module. So if we were to place this into an existing module I would propose to place it into the selection module as opposed to the groups module. On the other hand, selection seems to be a bit of a dumping ground for UI state and I'm not sure if it's a good idea to keep dumping things in there. I would like this to be easy to find, and give direct-linked group things a place to live (something that has come up a couple times already is reporting more specific error messages and I could see the need for that kind of information to be stored in the store but I don't want to get too ahead of myself) so putting it into it's own module paves the way for that. The groups are data whereas the error state is really a UI concern and those feel like separate things to me.